package.json

```diff
@@ -36,9 +36,12 @@
     "dateformat": "^3.0.3",
     "fast-safe-stringify": "^2.0.7",
     "jmespath": "^0.15.0",
+    "joycon": "^2.2.5",
     "pump": "^3.0.0",
     "readable-stream": "^3.4.0",
-    "split2": "^3.1.1"
+    "rimraf": "^2.6.3",
```

Comment:
Why is this a regular dependency?

Reply:
You are right, failed to spot it. Will change to dev dependency.